### ISSUE TITLE
Commonsense QA Training accuracy being 20% always 

### ISSUE THREAD

**Issue by yuchenlin** [pytorch/fairseq#1687]
## 🐛 Bug

The example script for fine-tuning roberta for Commonsense QA does not work correctly. The training accuracy is always about 20%.  This is the same issue with #1143 and #1121 , which were both closed but without solutions. 

### To Reproduce

Steps to reproduce the behavior (**always include the command you ran**):

1. Run cmd '....'

Just exactly follow the [example].  *Please also try download the data with the script again.*(https://github.com/pytorch/fairseq/tree/master/examples/roberta/commonsense_qa)** script with lateset fairseq and pytorch 1.2.0.

2. See error
The training acc is always about 20%, even after many many steps. 

```
2020-02-08 15:19:59 | INFO | train | epoch 002:    174 / 609 loss=2.327, nll_loss=0.097, accuracy=20, wps=910.8, ups=2.37, wpb=384.9, bsz=16, num_updates=
784, lr=7.77544e-06, gnorm=2.027, clip=0, oom=0, loss_scale=128, train_wall=55, ppl=1.07, wall=277
2020-02-08 15:20:07 | INFO | train | epoch 002:    199 / 609 loss=2.327, nll_loss=0.097, accuracy=19.7, wps=938.8, ups=2.44, wpb=384.7, bsz=16, num_update
s=809, lr=7.68772e-06, gnorm=2.013, clip=0, oom=0, loss_scale=128, train_wall=63, ppl=1.07, wall=285
2020-02-08 15:20:15 | INFO | train | epoch 002:    224 / 609 loss=2.327, nll_loss=0.097, accuracy=19.7, wps=962.8, ups=2.5, wpb=385, bsz=16, num_updates=8
34, lr=7.6e-06, gnorm=1.999, clip=0, oom=0, loss_scale=128, train_wall=71, ppl=1.07, wall=293
2020-02-08 15:20:23 | INFO | train | epoch 002:    249 / 609 loss=2.327, nll_loss=0.097, accuracy=19.7, wps=980.9, ups=2.55, wpb=384.9, bsz=16, num_update
s=859, lr=7.51228e-06, gnorm=1.991, clip=0, oom=0, loss_scale=128, train_wall=79, ppl=1.07, wall=301
2020-02-08 15:20:32 | INFO | train | epoch 002:    274 / 609 loss=2.326, nll_loss=0.096, accuracy=19.8, wps=998.5, ups=2.59, wpb=386.1, bsz=16, num_update
s=884, lr=7.42456e-06, gnorm=1.981, clip=0, oom=0, loss_scale=128, train_wall=87, ppl=1.07, wall=310
2020-02-08 15:20:40 | INFO | train | epoch 002:    299 / 609 loss=2.326, nll_loss=0.096, accuracy=20, wps=1012.1, ups=2.62, wpb=385.8, bsz=16, num_updates
=909, lr=7.33684e-06, gnorm=1.972, clip=0, oom=0, loss_scale=128, train_wall=95, ppl=1.07, wall=318
2020-02-08 15:20:48 | INFO | train | epoch 002:    324 / 609 loss=2.326, nll_loss=0.096, accuracy=20, wps=1020.4, ups=2.65, wpb=385.6, bsz=16, num_updates
=934, lr=7.24912e-06, gnorm=1.97, clip=0, oom=0, loss_scale=128, train_wall=103, ppl=1.07, wall=326
2020-02-08 15:20:56 | INFO | train | epoch 002:    349 / 609 loss=2.326, nll_loss=0.097, accuracy=20.2, wps=1030.8, ups=2.68, wpb=385.3, bsz=16, num_updat
es=959, lr=7.1614e-06, gnorm=1.975, clip=0, oom=0, loss_scale=128, train_wall=111, ppl=1.07, wall=334
2020-02-08 15:21:04 | INFO | train | epoch 002:    374 / 609 loss=2.326, nll_loss=0.096, accuracy=20.3, wps=1039.3, ups=2.7, wpb=385.4, bsz=16, num_updates=984, lr=7.07368e-06, gnorm=1.976, clip=0, oom=0, loss_scale=128, train_wall=119, ppl=1.07, wall=342                                                     2020-02-08 15:21:12 | INFO | train | epoch 002:    399 / 609 loss=2.326, nll_loss=0.097, accuracy=20, wps=1048.5, ups=2.72, wpb=384.8, bsz=16, num_updates=1009, lr=6.98596e-06, gnorm=1.973, clip=0, oom=0, loss_scale=128, train_wall=127, ppl=1.07, wall=350
```

### Environment

 - fairseq Version (e.g., 1.0 or master): master
 - PyTorch Version (e.g., 1.0) 1.2.0
 - OS (e.g., Linux): linux ubuntu 
 - How you installed fairseq (`pip`, source): source 
 - Build command you used (if compiling from source): `pip install --editable .`
 - Python version: 3.7
 - CUDA/cuDNN version: 10.0
 - GPU models and configuration: 1080Ti
 - Any other relevant information:

**Comment by yuchenlin** [comment]
Hi @myleott , hope you could help on this please!

**Comment by yuchenlin** [comment]
I figured it out. the script is not using torch hub and has to set the local folder of roberta manually

**Comment by myleott** [comment]
Can you include the beginning of the log, including the Namespace with the args that you passed?

**Comment by yuchenlin** [comment]
Hi @myleott  thanks for the quick reply! I have figured it out. The script does work after I redownload the robert.large files, and create a totally new conda env for this.  Not sure about the exact reason about this but I guess it may be because of my previous env having some bugs.
